examples/website-demos/social-messaging/src/components/MessageInput/SocialMessageInput.tsx

```diff
@@ -115,27 +115,29 @@ export const SocialMessageInput = (props: Props) => {
             giphyState
           }
         >
-          <EmojiPicker />
           <div className={`input-ui-input ${giphyState ? 'giphy' : ''}`}>
             {giphyState && !numberOfUploads && <GiphyIcon />}
             <UploadsPreview />
-            <ChatAutoComplete onChange={onChange} placeholder='Send a message' />
-            {
-              <>
-                {!giphyState && (
-                  <div
-                    className='input-ui-input-emoji-picker'
-                    ref={emojiPickerRef}
-                    onClick={openEmojiPicker}
-                  >
-                    <EmojiPickerIcon />
-                  </div>
-                )}
-              </>
-            }
+            <div className='input-ui-wrapper-emoji'>
```

Comment:
this classname doesn't make a ton of sense, in fact i think we can probably clean up the naming higher up the tree as well. for instance we have `input-ui-container` at the outermost level and then `input-ui-wrapper` nested on level within. outermost should probably be `input-ui` and then we append semantic naming related to their function after, so the first nested div would be something like `input-ui-icons`, which makes more sense than wrapper inside container